repo/packages/M/marathon-lb/19/marathon.json.mustache

```diff
@@ -9,7 +9,15 @@
     "docker": {
       "image": "{{resource.assets.container.docker.marathon-lb-docker}}",
       "network": "HOST",
-      "privileged": true
+      "privileged": true,
+      "parameters": [
+      {{#parameters}}
+        {
+          "key": "{{key}}",
+          "value": "{{value}}"
+        }{{^last}},{{/last}}
```

Comment:
@nlsun No, I couldn't find a more glamorous way to do this.  Basic moustache doesn't have an "@last" check.  Sorry - this is as clean as you're going to get it sadly.